common-arcana.lic

```diff
@@ -458,8 +488,20 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, data['tattoo_tm'])
-    prepare_time = Time.now
+    if data['runestone_name']
+      if !prepare_to_cast_runestone(data, settings)
+        return
+      end
+      if data['runestone_tm']
```

Comment:
I don't see how the two calls vary (except for the presence of `runestone_tm` which would already be true or false). Why is this conditional here?

Reply:
I could not get this too work any other way. I need to have the explicit if data['runestone_tm'] call.

Reply:
What I'm suggesting is to always have `data['runestone_tm']` in the call so these two lines are combined.

Reply:
Changed:
```ruby
if !prepare_to_cast_runestone(data, settings)
        return
      end
      return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'], data['prep_type'], tattoo_tm = false, data['runestone_name'], data['runestone_tm'])
```